tactics/class_tactics.ml

```diff
@@ -983,7 +986,9 @@ module Search = struct
 
   let typeclasses_resolve env evd debug depth unique p =
     let db = searchtable_map typeclasses_db in
-    typeclasses_eauto env evd ?depth unique (Hint_db.transparent_state db) [db] p
+    let st = Hint_db.transparent_state db in
+    let modes = Hint_db.modes db in
+    typeclasses_eauto env evd ?depth unique (modes,st) [db] p
 end
 
 (** Binding to either V85 or Search implementations. *)
```

Comment:
That's not related to this PR but this is an outdated comment that could be removed.

Reply:
Removed.